### ISSUE TITLE
Cumulative downloads chart

### ISSUE THREAD

**Issue by benmccann** [pvorb/npm-stat.com#106]
I'd love to see a chart that shows the total number of downloads as a running total instead of just the per-week or per-month numbers

**Comment by pvorb** [comment]
The problem with that is that this piece of software has to rely on npm's API, which only delivers a count for each day. And npm doesn't store these numbers forever. So I only could provide the running total for the provided time frame.

**Comment by benmccann** [comment]
That'd still be pretty cool though. Especially in my case where I want to link to it for a library that's only a year old, so that basically would cover the whole time since inception

**Comment by pvorb** [comment]
If you lib is that new, yoi can just look at the current year or the last two years and that's your total, right?

**Comment by benmccann** [comment]
yeah. I wanted a chart with cumulative running total instead of weekly/month numbers

I ended up building it myself: https://kit-stats.netlify.app/. I still think it'd be a cool feature for npm-stat.com though :smile:

**Comment by csandman** [comment]
I also came here to leave an issue about this, I think this would be a great feature.  Even if it was just a running total of the values in the range selected, I think it would be cool to see total downloads in terms of a curve!
